test/Microsoft.AspNet.Mvc.Core.Test/Rendering/DefaultEditorTemplatesTest.cs

```diff
@@ -808,6 +809,21 @@ public ViewDataDictionary ViewData
                 get { return _innerHelper.ViewData; }
             }
 
+            public IHtmlEncoder HtmlEncoder
```

Comment:
@NTaylorMullen this is a test `IHtmlHelper` implementation.  the properties must exist or the compiler complains.